server/services/deployment.ts

```diff
@@ -5,13 +5,20 @@ import DeploymentModel from '../models/Deployment'
 import { Deployment, User, ModelId } from '../../types/interfaces'
 import AuthorisationBase from '../utils/AuthorisationBase'
 import { asyncFilter } from '../utils/general'
+import { SerializerOptions } from '../utils/logger'
 
 const authorisation = new AuthorisationBase()
 
 interface GetDeploymentOptions {
   populate?: boolean
 }
 
+export function serializedDeploymentFields(): SerializerOptions {
+  return {
+    mandatory: ['_id', 'uuid', 'name', 'model'],
```

Comment:
Here `model` might be an ObjectID or a model, to handle this you should put this under the other `serializable` option, look in `version.ts` for an example.

Reply:
Resolved